views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,4 @@
+    <a href="/random-beer"> <img src="{{image_url}}" alt="img"> </a>
```

Comment:
Right now when you click on a beer in the beers list, you get taken to a random beer each time (instead of to that particular's beer page via /{{_id}}. It's fine as it was a bonus iteration, but it's probably better to leave the link blank then to take the user to the wrong page. But I know you understand this :)